atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/GeneratedImport.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.table.description.render;
+
+import com.squareup.javapoet.ClassName;
+import javax.lang.model.SourceVersion;
+
+/**
+ * From autovalue's GeneratedImport class
+ */
+final class GeneratedImport {
+    /**
+     * Returns the qualified name of the {@code @Generated} annotation available during a compilation
+     * task.
+     */
+    static String generatedAnnotationType() {
+        return SourceVersion.latestSupported().compareTo(SourceVersion.RELEASE_8) > 0
+                ? "javax.annotation.processing.Generated"
+                : "javax.annotation.Generated";
+    }
+
+    static ClassName generatedAnnotationClassName() {
+        return SourceVersion.latestSupported().compareTo(SourceVersion.RELEASE_8) > 0
+                ? ClassName.get("javax.annotation.processing", "Generated")
+                : ClassName.get("javax.annotation", "Generated");
+    }
```

Comment:
AtlasDB moved to JDK 11 in https://github.com/palantir/atlasdb/pull/5501 over a year ago, so could we just use `javax.annotation.processing.Generated` directly and kill off references to `javax.annotation.Generated`? I imagine that would simplify a lot of these bits.

https://github.com/palantir/atlasdb/blob/3c160a1cbef50bd45c4de50e5349158cd3e4542b/build.gradle#L39-L40

Reply:
https://github.com/palantir/atlasdb/pull/6202 for what I was envisioning (builds on commit from https://github.com/palantir/atlasdb/pull/6201 to fix `generateSchemas`)

Reply:
Ah, so I don't have to support any form of backwards compatibility here?